diff --git a/node_modules/detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/ReactNativeIdlingResources.kt b/node_modules/detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/ReactNativeIdlingResources.kt
index 48335c0..487cb90 100644
--- a/node_modules/detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/ReactNativeIdlingResources.kt
+++ b/node_modules/detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/ReactNativeIdlingResources.kt
@@ -111,6 +111,7 @@ class ReactNativeIdlingResources(
         loopers.clear()
     }
 
+    @OptIn(kotlin.ExperimentalStdlibApi::class)
     private fun unregisterIdlingResources() {
         IdlingResourcesName.entries.forEach {
             removeIdlingResource(it)
